cordapp/src/main/kotlin/net/corda/plugins/CordappPlugin.kt

```diff
@@ -156,18 +156,13 @@ class CordappPlugin : Plugin<Project> {
         return Pair(targetPlatformVersion, minimumPlatformVersion)
     }
 
-    private fun parseVersion(versionStr: String?): Int {
-        if (versionStr == null)
+    private fun checkCorDappVersionId(versionId: Int?): Int {
+        if (versionId == null)
             throw InvalidUserDataException("Target versionId not specified. Please specify a whole number starting from 1.")
```

Comment:
Ugh, these error messages don't even contain the word "cordapp"! How is the user supposed to know what has gone wrong without any context?